### DISCUSSION TITLE
Research needed on applicability of KGE to heads

### DISCUSSION THREAD

**Discussion by raoulcollenteur** [pastas/pastas#712] [Ideas]
Although now commonly used in rainfall-runoff modeling, the KGE is still new to evaluate head time series. Although often the results look reasonable, it is not uncommon that the value of the KGE does not at all match the visual interpretation. So:

1. a warning that the KGE should be used with caution
2. more research is needed to see if it really is that applicable, and which form (https://github.com/pastas/pastas/issues/606)

I still think the idea of such a metric that combines different types of errors is usefull, but perhaps the KGE (as we haveit now), is not the best. If anyone has time/a project to look into this I'll be happy to help!

Cheers
Raoul

ps. I'll add an example later

**Comment by martinvonk** [comment]
Came across this article today with uses KGE with KMoments; [Advancing objective functions in hydrological modelling: Integrating knowable moments for improved simulation accuracy - Alonso Pizarro, Joaquín Jorquera](https://doi-org.tudelft.idm.oclc.org/10.1016/j.jhydrol.2024.131071).  This one is also for streamflow but might be interesting.
